### PR TITLE
make timers constructable and deconstructable

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
similar recipe to wallmount gens
- 5 steel 2 lv and a board
- for screen and brig timers, some glass as well
- bada bing bada boom

only non-mapped timers can be deconstructed, similar to wallmount gens.
i tried to make it so all of them can be, but mapped ones would never have a board no matter what i tried so yeah

the board is unlocked with electrical engineering and can be made at protolathes and circuit imprinters

**Media**

constructing a signal timer, converting it to brig timer, then to screen timer.

https://user-images.githubusercontent.com/39013340/233313463-9fef8581-67aa-4c1e-acd3-b38e438e7cc1.mp4

deconstructing a (non-mapped) timer (that had board removed prior)

https://user-images.githubusercontent.com/39013340/233313611-abbd3eda-5403-4cd2-bcf4-9684d51133d4.mp4

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added construction for signal timer, screen timer and brig timer.

### CODE REVIEW

**Comment by deltanedas** [comment]
@EmoGarbage404 any issues with this or is it good

File: Resources/Prototypes/Entities/Structures/Wallmounts/timer.yml [hunk 1]
```diff
@@ -61,3 +67,33 @@
...
+  name: timer frame
+  description: A construction frame for a timer.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Physics
+    canCollide: false
+  - type: Fixtures
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Wallmounts/timer.yml:88]: Remove.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Review by EmoGarbage404 — APPROVED** [review]
Looks fine at a glance. Assuming it works in game, gtg

**Comment by deltanedas** [comment]
@EmoGarbage404 j

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**